README.md

```diff
@@ -43,7 +43,7 @@ Features
 
 If this feature list left you scratching your head, you can first read
 more about the [Vector Space Model] and [unsupervised document analysis]
-on Wikipedia.
+on Wikipedia.    
```

Comment:
In the future I'd suggest leaving out, or removing at the end, unnecessary whitespace changes in documentation.